src/analyzer/protocol/dns/DNS.h

```diff
@@ -463,11 +508,15 @@ constexpr auto TYPE_EDNS [[deprecated("Remove in v4.1. Use zeek::analyzer::dns::
 constexpr auto TYPE_TKEY [[deprecated("Remove in v4.1. Use zeek::analyzer::dns::detail::TYPE_TKEY.")]] = zeek::analyzer::dns::detail::TYPE_TKEY;
 constexpr auto TYPE_TSIG [[deprecated("Remove in v4.1. Use zeek::analyzer::dns::detail::TYPE_TSIG.")]] = zeek::analyzer::dns::detail::TYPE_TSIG;
 constexpr auto TYPE_CAA [[deprecated("Remove in v4.1. Use zeek::analyzer::dns::detail::TYPE_CAA.")]] = zeek::analyzer::dns::detail::TYPE_CAA;
+constexpr auto TYPE_SSHFP [[deprecated("Remove in v4.1. Use zeek::analyzer::dns::detail::TYPE_SSHFP.")]] = zeek::analyzer::dns::detail::TYPE_SSHFP;
```

Comment:
Same as above about not needing to add the deprecated versions.

Reply:
Removed them from this file as well.